deployment/helm/event-sourcing-microservices-example/charts/edge-service

```diff
@@ -0,0 +1 @@
+/home/pczarkowski/development/scratch/event-sourcing-microservices-example/deployment/helm/edge-service
```

Comment:
@paulczar This doesn't look right. Can you x2 check?

Reply:
oh dumb symlink bs! will fix.